configuration/peloton.conf

```diff
@@ -0,0 +1,119 @@
+# -----------------------------
```

Comment:
It is loaded at runtime like this:

`./bin/peloton -flagfile ../configuration/peloton.conf`